### ISSUE TITLE
[Feature Request] New plugin `remove-pwa` 

### ISSUE THREAD

**Issue by Mister-Hope** [vuepress-theme-hope/vuepress-theme-hope#2452]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

A plugin to remove pwa quickly.

**Comment by Mister-Hope** [comment]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/7b3e798ec599068975636b2400cfdd75bcabd070

**Comment by hschaefer123** [comment]
Unbelievable fast 😉

I just  saw your plugin commit.

Will be back from Switzerland and try your code directly on monday.
Thank you very much.

Regards Holger

**Comment by hschaefer123** [comment]
Hi Mr Hope,

i just deployed a new version of
https://btp.udina.de

that is using your pwa remover and seems to work like a charm.

It was a little bit problematic, because there was a dependence to beta.53 and i was using beta.49 and webpack.
I installed a fresh new version of beta.53 and now i am also able to use vite (formerly always throwing issues for me).

So great, that i am now on latest release and also be able to use .53.

For your info concerning plugin. 
There are some wrong texts inside docs (i think you cloned pwa plugin).

I had to change

```ts
import { removePWAPlugin } from "vuepress-plugin-remove-pwa";

and also

plugins: [
  removePWAPlugin({}),    
]
```

just replacing ````pwaPlugin``` by ```removePWAPlugin```

Thanks a lot for offering this plugin and quite fast.

Best regards
Holger
